src/blocks/pacman.rs

```diff
@@ -264,7 +266,7 @@ fn get_pacman_available_updates() -> Result<String> {
             .env("LC_ALL", "C")
             .args(&[
                 "-c",
-                &format!("fakeroot pacman -Qu --dbpath \"{}\"", updates_db),
+                &format!("pacman -Qu --dbpath \"{}\"", updates_db),
```

Comment:
Why remove fakeroot here?

Reply:
Instead of calling it through a shell which brings no advantage to my knowledge

Reply:
Oh sorry I missed that fakeroot was moved to the other function. That function is kinda useless since it's only used in one place but that's another issue.